packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -5787,6 +5787,41 @@ export class WalletService implements IWalletService {
     });
   }
 
+  sardineGetSupportedTokens(req): Promise<any> {
+    return new Promise((resolve, reject) => {
+      const keys = this.sardineGetKeys(req);
+      const API = keys.API;
+      const CLIENT_ID = keys.CLIENT_ID;
+      const SECRET_KEY = keys.SECRET_KEY;
+
+      const secret = `${CLIENT_ID}:${SECRET_KEY}`;
+      const secretBase64 = Buffer.from(secret).toString('base64');
+
+      const headers = {
+        'Content-Type': 'application/json',
+        'Authorization': `Basic ${secretBase64}`,
+      };
+
+      const URL: string = API + '/v1/supported-tokens';
+
+      this.request.post(
+        URL,
+        {
+          headers,
+          body: req.body,
```

Comment:
This method does not requiere body parameters
https://docs.sardine.ai/docs/integrate-payments/on-ramp-api/operations/list-supported-tokens